### PR TITLE
Fixes user dashboard does not handle the null array appropriately

### PR DESCRIPTION
**Pull request by qhuynh96**

### CODE REVIEW

File: prow/cmd/deck/static/user-script.js [hunk 1]
```diff
@@ -41,6 +41,10 @@ function loadUserDashboard() {
         });
     });
     const container = document.querySelector("#main-container");
+    if (!userData.PullRequests || userData.PullRequests.length === 0) {
+        container.appendChild(document.createTextNode("You don't have any open PR"));
```
Comment (BenTheElder) [inline prow/cmd/deck/static/user-script.js]: I would say "No open PRs found". The current text will be very frustrating if the data isn't loading correctly :-)

Comment (qhuynh96) [reply prow/cmd/deck/static/user-script.js]: done

File: prow/cmd/deck/static/user-script.js [hunk 2]
```diff
@@ -461,7 +466,8 @@ function createPRCardBody(pr, builds, queries) {
     cardBody.classList.add("mdl-card__supporting-text");
     cardBody.appendChild(title);
     cardBody.appendChild(createJobStatus(builds));
-    cardBody.appendChild(createMergeStatus(pr.Labels.Nodes, queries));
+    const nodes = pr.Labels && pr.Labels.Nodes ? pr.Labels.Nodes : [];
```
Comment (stevekuznetsov) [inline prow/cmd/deck/static/user-script.js]: Not the biggest fan of ternary operators and one liners like this, they're a bit hard to read. Style nit though, so meh.

/lgtm
/hold cancel
/woof

**Comment by BenTheElder** [comment]
also normally would squash a commit like "wording" but :man_shrugging:

**Pull request merged**